src/testdir/test_spellrare.vim

```diff
@@ -0,0 +1,65 @@
+" Test spell checking
+
+source check.vim
+CheckFeature spell
+
+func TearDown()
+  set nospell
```

Comment:
At the end of the Test_spellrareword() function, the 'spell' option is reset.  So this is not needed in TearDown().